portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase

```diff
@@ -0,0 +1,86 @@
+@component-name = "portal-analytics-cloud"
+definition {
+
+	property analytics.cloud.enabled = "true";
+	property minimum.slave.ram = "32";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Analytics Cloud";
+
+	setUp {
+		ACUtils.launchAC();
+
+		ACSettings.takeTokenFromAnalyticsCloud();
+
+		TestCase.setUpPortalInstance();
+
+		ACUtils.launchDXP();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		ACUtils.tearDownDXP();
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+
+		ACUtils.tearDownAC();
+	}
+
+	@description = "Search for specific segment"
+	@priority = "5"
+	test CanSearchField {
```

Comment:
hi @calvinctvo , I would suggest you rename the test to `CanSearchForSegment`, because it is basically what you are doing here.

And then update the test case summary on the Test Map to `Search for a segment on the segment list`.